### PR TITLE
Added SourcePath to CefSharp.Core and CefSharp.BrowserSubprocess.Core

### PR DESCRIPTION
**Pull request by merceyz**
- Added `SourcePath` for CEF files to `CefSharp.Core` and `CefSharp.BrowserSubprocess.Core`

Without this it compiles perfectly fine but `IntelliSense` doesn't suggest anything for CEF related types. They all show up as `error-type`.

### CODE REVIEW

**Comment by amaitland** [comment]
`Intellisense` works for me without this change. It can be a little hit and miss. It's usually only a problem when doing a clean checkout, needs a restart after a successful build.

Do you have a reference doc for `IncludePath`?

**Comment by merceyz** [comment]
I just tried: clean checkout, restored nuget packages, build all, restart.
Intellisense still doesn't show anything, with include path set it works instantly.

I don't, no

**Comment by amaitland** [comment]
intellisense1
intellisense2

As it stands this is what I get using `master`, I pretty much get what I would expect. 

When I have a few minutes I'll see what I can find on `IncludePath`. We currently have `AdditionalIncludeDirectories` set which has basically the same value. So I'm curious as to what the difference is.

**Comment by merceyz** [comment]
https://stackoverflow.com/a/6883392/2923845
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2008/t9az1d21(v=vs.90)
https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2010/73f9s62w(v=vs.100)

Seems like `AdditionalIncludeDirectories` is for the compiler specifically and that `IncludePath` is for general use

File: CefSharp.BrowserSubprocess.Core/CefSharp.BrowserSubprocess.Core.vcxproj [hunk 1]
```diff
@@ -73,25 +73,29 @@
     <IgnoreImportLibrary>true</IgnoreImportLibrary>
     <OutDir>bin\$(Platform)\$(Configuration)\</OutDir>
     <IntDir>obj\$(Platform)\$(Configuration)\</IntDir>
+    <IncludePath>$(SolutionDir)packages\$(CefSdkVer)\CEF;$(IncludePath)</IncludePath>
```
Comment (amaitland) [inline CefSharp.BrowserSubprocess.Core/CefSharp.BrowserSubprocess.Core.vcxproj]: Does using `SourcePath` work for you? Description given in `VS` being `Path to use when searching for source files to use for Intellisense`

```
<SourcePath>$(SolutionDir)packages\$(CefSdkVer)\CEF;$(SourcePath)</SourcePath>
```

Comment (merceyz) [reply CefSharp.BrowserSubprocess.Core/CefSharp.BrowserSubprocess.Core.vcxproj]: Yes, same effect.
Should probably switch over to that one then as that's what it's made for

**Comment by amaitland** [comment]
Thanks :+1:

**Pull request merged**